portal-frontend/src/app/services/public/public-application.dto.ts

```diff
@@ -4,6 +4,7 @@ import {
   NaruSubtypeDto,
   ProposedLot,
 } from '../application-submission/application-submission.dto';
+import { CovenantTransfereeDto, OwnerTypeDto } from '../covenant-transferee/covenant-transferee.dto';
```

Comment:
Nit: Were the two new imports used anywhere?

Reply:
Good catch, nothing gets passed Sandra 🥅